### PR TITLE
[TW-960] Doc Auto update/refresh of OAuth2 access tokens

### PR DESCRIPTION
**Pull request by mark-anthony-postman**
Noted that the **Auto-refresh token** toggle and manual **Refresh** option are available only when a refresh token is present

### CODE REVIEW

**Review by MatthewPinsker-postman — APPROVED** [review]
Looks good!

**Comment by Tegimufranka9639** [comment]
Franklyn mutegi kunganga

**Comment by Tegimufranka9639** [comment]
> Noted that the **Auto-refresh token** toggle and manual **Refresh** option are available only when a refresh token is present

tw-960-refresh-token

**Comment by Tegimufranka9639** [comment]
Franklyn mutegi kunyanga

**Comment by Tegimufranka9639** [comment]
Access gh pr checkout 4600

**Comment by Tegimufranka9639** [comment]
Add auto refresh postman API

**Review by Tegimufranka9639 — COMMENTED** [review]
Agreed 💯

**Comment by Tegimufranka9639** [comment]
Patch

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Review by abhijeetborole — APPROVED** [review]

File: src/pages/docs/sending-requests/authorization.md [hunk 1]
```diff
@@ -339,13 +339,13 @@ If authentication fails or times out, Postman will display an error message. You
 
 Before an OAuth 2.0 token generated in Postman expires, Postman automatically refreshes it in the background before you send a request that uses it. The refreshed access token is updated in any requests that it's used in. Auto-refreshing is the default behavior.
 
-To turn this feature off or on, select **Auto-refresh access token**.
+To turn this feature off or on, select **Auto-refresh access token**. To manually refresh a token, select **Refresh**. The token's expiration time displays if the token expires within the next day.
 
-<img alt="Auto-refresh an OAuth 2.0 access token" src="https://assets.postman.com/postman-docs/v10/authorization-oauth2-auto-refresh-v10.jpg" width="500px"/>
+Auto-refresh is available when a refresh token is present. If no refresh token is present, the **Auto-refresh access token** toggle and the manual **Refresh** option aren't available. To check if a refresh token is present, select **Manage Tokens** in the **Token** dropdown list. To be present, a refresh token must be sent back by the service you are using. Postman doesn't provide refresh tokens.
```
Comment (giridharvc7) [inline src/pages/docs/sending-requests/authorization.md:344]: Can we change this to something like -
"To check if a refresh token is present, select **Manage Tokens** in the **Token** dropdown list. If refresh token is not present, check with the Auth service. Postman will not be able to refresh the access tokens without the refresh token"

**Pull request merged**